extensions/blocks/eventbrite/eventbrite.php

```diff
@@ -47,14 +75,62 @@ function render_block( $attr, $content ) {
 	);
 
 	$widget_id = wp_unique_id( 'eventbrite-widget-' );
-
-	wp_enqueue_script( 'eventbrite-widget', 'https://www.eventbrite.com/static/widgets/eb_widgets.js', array(), JETPACK__VERSION, true );
-
-	// Add CSS to hide direct link.
-	Jetpack_Gutenberg::load_assets_as_required( FEATURE_NAME );
+	$is_amp    = is_amp_request();
 
 	// Show the embedded version.
 	if ( empty( $attr['useModal'] ) && ( empty( $attr['style'] ) || 'modal' !== $attr['style'] ) ) {
+		return render_embed_block( $widget_id, $is_amp, $attr );
+	} else {
+		return render_modal_block( $widget_id, $is_amp, $attr, $content );
+	}
+}
+
+/**
+ * Render block with embed style.
+ *
+ * @param string $widget_id Widget ID to use.
+ * @param bool   $is_amp    Whether AMP page.
+ * @param array  $attr      Eventbrite block attributes.
+ * @return string Rendered block.
+ */
+function render_embed_block( $widget_id, $is_amp, $attr ) {
+
+	// $content contains a fallback link to the event that's saved in the post_content.
+	// Append a div that will hold the iframe embed created by the Eventbrite widget.js.
+	$classes = Jetpack_Gutenberg::block_classes( FEATURE_NAME, $attr );
```

Comment:
Once you rebase (see my comment above), this should be `Blocks::classes()`. Same below.

Reply:
Done!